server/src/test/java/org/opensearch/index/shard/RemoteStoreRefreshListenerTests.java

```diff
@@ -322,11 +322,12 @@ public void testRefreshSuccessOnThirdAttemptAttempt() throws Exception {
         assertBusy(() -> assertEquals(0, successLatch.getCount()));
         RemoteRefreshSegmentPressureService pressureService = tuple.v2();
         RemoteRefreshSegmentTracker segmentTracker = pressureService.getRemoteRefreshSegmentTracker(indexShard.shardId());
-        assertEquals(0, segmentTracker.getBytesLag());
-        assertEquals(0, segmentTracker.getRefreshSeqNoLag());
-        assertEquals(0, segmentTracker.getTimeMsLag());
-        assertEquals(2, segmentTracker.getTotalUploadsFailed());
-
+        assertBusy(() -> {
```

Comment:
Do we need similar change for other tests as well? `testRefreshSuccessOnSecondAttempt` is mentioned https://github.com/opensearch-project/OpenSearch/issues/8279 as flaky. Maybe the same issue?

Also, can we please fix the test name? `testRefreshSuccessOnThirdAttemptAttempt` -> `testRefreshSuccessOnThirdAttempt`